### DISCUSSION TITLE
Accessibility for multiple selection control is not working in mobile.

### DISCUSSION THREAD

**Discussion by kidtalkative** [microsoft/fluentui#28444] [Q&A]
We're facing an issue about a11y for multiple selection control in mobile.
**For Android - Talkback in Android 13**
We're able to open the list. Talkpad announces the number of items but we cannot navigate to the list nor select items. Try to swipe the screen will focus on the next item on the page instead of traversing the list.

**For IOS - VoiceOver in IOS 16.5.1**
We're able to open and select multiple items in IOS. However, there's no way to remove selected items. VoiceOver keeps announcing using the delete button or the back button to remove the selected item. Try to swipe or double tap leads to nothing.

**Comment by behowell** [comment]
Hi @kidtalkative, thanks for the report. Please log this as a bug so we can investigate and track a fix. Choose "Bug Report" from here: https://github.com/microsoft/fluentui/issues/new/choose.  The most important thing is to include a link to a repro (e.g. in codesandbox or similar) in your bug report so we can investigate. Thanks!
